package.json

```diff
@@ -1,6 +1,7 @@
 {
   "private": true,
   "devDependencies": {
-    "ws": "~0.4.28"
+    "ws": "~0.4.28",
+    "html-minifier-terser": "5.0.2"
```

Comment:
The reason `ws` is a "dev" depenedency is because its only need to run test code IIRC.

I think `"html-minifier-terser` should be a regular depedency since all users will want it.

Reply:
In https://medium.com/@dylanavery720/npmmmm-1-dev-dependencies-dependencies-8931c2583b0c it says

```
The difference between these two, is that devDependencies are modules which are only required during development, while dependencies are modules which are also required at runtime.
```

I understood the runtime modules to be ones that get to be a part of the final built page when you do `npm build`? Since we are not doing that because this is not a web site project, I thought all our dependencies will be dev dependencies?

Reply:
devDependencies are ones that are needed by people who are actually working the the module in questions: emscripten.  i.e. they are only needed by people like you and I who work on emscripten.

Users of the module get the only the regular dependencies.    If we ever did package emscripten in npm for example most users wouldn't want `ws` but they all would want `closure`.

Reply:
Oh gotcha.. sometimes first link on google is not the right one.. https://docs.npmjs.com/files/package.json#devdependencies explains it proper. Updated.